### ISSUE TITLE
Test additional scenarios of SMT creation.

### ISSUE THREAD

**Issue by Kiwonik** [steemit/steem#1729]
SMT creation needs additional scenarios being tested as part of its unit tests. These are:
1. (Negative) Insufficient balance of control account vs chosen currency (as set in `smt_creation_fee` field of `smt_create_operation`). This is partially done (TESTS vs TBD).
1. (Negative) Too low fee declared in `smt_creation_fee` field of `smt_create_operation` as compared with the value stored in `dynamic_global_properties_object`. Check both allowed currencies.

**Comment by Kiwonik** [comment]
Blocked by #1738

**Comment by Kiwonik** [comment]
@youkaicountry This issue builds upon the code of almost complete and recently unblocked issue #1653 
I'm adding the latter to project 5 (SMT) for clarity.

**Comment by youkaicountry** [comment]
@kiwonik Okay, this is going to be lower priority for now than the other issues already present on the board. I'll move it to the bottom of the stack for now, please prioritize other issues for now.
